test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,175 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { expect } from '@playwright/test';
+import { Application, Logger, PositronPythonFixtures, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+
+/*
+ *  Welcome test cases.
+ */
+
+export function setup(logger: Logger) {
+	describe('Welcome Page', () => {
+		installAllHandlers(logger);
+
```

Comment:
For my system, I was able to get past my issue of wrong interpreter getting selected by adding in a before to startup the python interpreter.  I think this is overall a good practice to ensure the desired interpreter is started and pushed to top of the list
```suggestion
		
		before(async function () {
			await PositronPythonFixtures.SetupFixtures(this.app as Application);
		});
		
```1

Reply:
I did notice on my AWS Workspace that it did take longer to scan for interpreters. Sounds like it would be good to get the Python interpreter selected earlier in the suite run before the Python tests run.